model_prices_and_context_window.json

```diff
@@ -21,6 +21,18 @@
         "supports_parallel_function_calling": true,
         "supports_vision": true
     },
+    "gpt-4o-latest": {
+        "max_tokens": 4096,
+        "max_input_tokens": 128000,
+        "max_output_tokens": 4096,
+        "input_cost_per_token": 0.0000025,
+        "output_cost_per_token": 0.00001,
+        "litellm_provider": "openai",
+        "mode": "chat",
+        "supports_function_calling": true,
+        "supports_parallel_function_calling": true,
+        "supports_vision": true
```

Comment:
This doesn't match with what I see in OpenAI's docs. The max tokens seems to be 16384 and not 4096 and the cost also doesn't match ($5 per million input tokens). https://platform.openai.com/docs/models/gpt-4o

Where did you get this data from?